porth.py

```diff
@@ -489,7 +490,7 @@ def generate_nasm_linux_x86_64(program: Program, out_file_path: str):
                     out.write("    cmovg rcx, rdx\n");
                     out.write("    push rcx\n")
                 elif op.operand == Intrinsic.LT:
-                    out.write("    ;; -- gt --\n")
+                    out.write("    ;; -- lt --\n")
```

Comment:
These look correct but out-of-scope for this change.

Maybe send them up as a separate PR?

Reply:
Yeah yeah no problem